src/pages/query-view/query-view.js

```diff
@@ -28,10 +29,10 @@ const QueryView = props => {
     }
   }, [shouldAutoExecuteQuery])
 
-  const columns = determineColumns(dataSources)
-  const data = getCleanData(queryData)
+  // const columns = determineColumns(dataSources)
+  // const data = getCleanData(queryData)
```

Comment:
just commenting in case these aren't supposed to be here

Reply:
They are not, I'll get rid of them